### DISCUSSION TITLE
Risk Per Trade

### DISCUSSION THREAD

**Discussion by huytradingz** [kernc/backtesting.py#1058] [Q&A] [answered]
Hi everyone. I would like to implement a 2% risk per trade for my strategy (risk 2% of my account per trade), and was wondering if backtesting.py includes options for us to set it, or that is something we have to calculate and implement it ourselves? If so, could I have an example of how it would be done? Would the code below be a potential approach?

`riskpertrade = 0.02 * self.equity`
`order_size = abs(int(riskpertrade / (price - stoploss_price)))`
`order_size_short = abs(int(riskpertrade / (price - stoploss_price_short)))`

Thank you!

**Comment by kernc** [comment, marked as answer]
Generally, this looks good. When the price breaks below your stop loss, your equity ought to be 2% lighter.
